ui/widgets/charts_panel.py

```
@@ -3,16 +3,83 @@
+            layout.addWidget(self._cpu_plot)
+            layout.addWidget(self._mem_plot)
+            self._bus.metric_sample.connect(self._update_charts)
+
+    def set_process(self, pid: int) -> None:
+        """Define o PID do processo alvo."""
+
+        self._pid = pid
+
+    def _toggle_metrics(self) -> None:
+        if self._collector:
+            asyncio.create_task(self._collector.stop())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)